MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
```

Comment:
We shouldn’t unconditionally remove the name from U.S. Routes. See https://github.com/mapbox/mapbox-navigation-ios/pull/291#discussion_r123060422. But I agree with this approach if we end up needing to remove the name from multiple networks.